.gitignore

```diff
@@ -45,3 +45,6 @@ vignettes/*.pdf
 # System files
 *.DS_Store
 
+# ignore database files
+*.db
+
```

Comment:
i guess it could be there, but the idea would be that you just run the scripts and it sets it up on your local machine without needing to live on github.